apps/asap-server/src/handlers/teams/index-handler.ts

```diff
@@ -0,0 +1,51 @@
+import { EventBridgeEvent } from 'aws-lambda';
+import algoliasearch, { SearchClient } from 'algoliasearch';
+import { TeamsEventType } from '../webhooks/webhook-teams';
+import Team, { TeamController } from '../../controllers/teams';
+import {
+  algoliaAppId,
+  algoliaIndexApiKey,
+  algoliaResearchOutputIndex,
+} from '../../config';
+import logger from '../../utils/logger';
+
+export const indexHandlerFactory = (
```

Comment:
I'd rename this one to `indexResearchOutputtByTeamHandler` and the other one to `indexResearchOutputHandler` or something along these lines cause it will be easier to use the named import this way